cms/utils/compat/forms.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
It should go away, because there's no need for it. No UTF-8 strings in the source code. (I know it's not part of the PR, but I like to clean up too when I touch a source file. It's just a comment, anyway.)